src/components/ColumnTitle/ColumnFilter.tsx

```diff
@@ -113,9 +121,14 @@ export const ColumnFilter: React.FC<ColumnFilterProps> = props => {
  */
 export function mapStateToProps(store: Store, ownProps: ColumnFilterOwnProps) {
     const widget = store.view.widgets.find(item => item.name === ownProps.widgetName)
+    const assocWidget = store.view.widgets.find(
```

Comment:
I think we should now move on to connecting components via [useSelector](https://react-redux.js.org/api/hooks#useselector) hook as its more backward compatibility friendly - `ColumnFilterProps` is a public interface and someone uses it the addition of mandatory `assocWidget` will crash the typings.
`<ColumnFilter />` is not realistically usable when not connected (internal use of `dispatch`), so connected props are also more of component internals then public API.